src/spikeinterface/qualitymetrics/misc_metrics.py

```diff
@@ -1365,3 +1365,68 @@ def _compute_rp_violations_numba(nb_rp_violations, spike_trains, spike_clusters,
             spike_train = spike_trains[spike_clusters == i]
             n_v = _compute_nb_violations_numba(spike_train, t_r)
             nb_rp_violations[i] += n_v
+
+
+def compute_sd_ratio(
+    wvf_extractor: WaveformExtractor,
+    censored_period_ms: float = 4.0,
+    correct_for_drift: bool = True,
+    unit_ids=None,
+    **kwargs,
+):
+    """
+    Computes the SD (Standard Deviation) of each unit spikes amplitude, and compare it to that of noise.
```

Comment:
```suggestion
    Computes the SD (Standard Deviation) of each unit's spike amplitudes, and compare it to the standard deviation of noise.
```

What is noise in this case? Might be helpful to say what noise you are using as the noise in the calculation.